documentation/faq/70-integrations.md

```diff
@@ -0,0 +1,15 @@
+---
+question: How do I use X with SvelteKit?
+---
+
+## How do I setup library X?
```

Comment:
Could you make these level 4 headings? I think will be the correct level for the site and it is easier to do it here than transforming the source.

Reply:
done. I don't quite understand why though. Are there level 2 and 3 headings coming from somewhere?

Reply:
Maybe level 3 is correct actually.

level 1 will be the page title which should be `svelte kit faq` or similar, level 2 will be the question itself (which is in the frontmatter), so maybe level 3 is correct for these subheadings.

Reply:
I wrote this in chat, this is the issue:"

> The whole heading level thing for docs is a little bit annoying. I think we should probably just use relatively normal heading levels and transform them when we format the docs. That way the readmes at least make sense for readers looking at them in source and we are free to mess around with those levels at any point without needing to rewrite anything.

Reply:
ok, updated to level 3